common/config/helpers.go

```diff
@@ -1,10 +1,21 @@
 package config
 
 import (
+	"strings"
+
 	"github.com/Masterminds/semver"
 	"github.com/pulumi/pulumi/sdk/v3/go/auto"
 )
 
+func FindEnvironmentName(environments []string, prefix string) string {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
This name could be more explicit: this function returns the first element in a list that is prefixed by a given prefix, it could become `FindEnvironmentNameByPrefix`

Reply:
To me, the parameters are part of the function signature (even though `Go` does not support overloading), so `FindEnvironmentNameByPrefix(prefix)` is redundant.